ironfish/src/network/peerNetwork.ts

```diff
@@ -190,23 +190,38 @@ export class PeerNetwork {
       (message) => this.onBlockRequest(message),
     )
 
-    this.registerHandler(
-      NodeMessageType.NewBlock,
-      RoutingStyle.gossip,
-      (p) => {
-        return this.chain.verifier.verifyNewBlock(p, this.node.workerPool)
-      },
-      (message) => this.onNewBlock(message),
-    )
+    if (this.node.config.get('enableSyncer')) {
+      this.registerHandler(
+        NodeMessageType.NewBlock,
+        RoutingStyle.gossip,
+        (p) => {
+          return this.chain.verifier.verifyNewBlock(p, this.node.workerPool)
+        },
+        (message) => this.onNewBlock(message),
+      )
 
-    this.registerHandler(
-      NodeMessageType.NewTransaction,
-      RoutingStyle.gossip,
-      (p) => {
-        return this.chain.verifier.verifyNewTransaction(p)
-      },
-      async (message) => await this.onNewTransaction(message),
-    )
+      this.registerHandler(
+        NodeMessageType.NewTransaction,
+        RoutingStyle.gossip,
+        (p) => {
+          return this.chain.verifier.verifyNewTransaction(p)
+        },
+        async (message) => await this.onNewTransaction(message),
+      )
+    } else {
+      this.registerHandler(
+        NodeMessageType.NewBlock,
+        RoutingStyle.gossip,
+        () => Promise.resolve(undefined),
+        () => Promise.resolve(undefined),
+      )
+      this.registerHandler(
+        NodeMessageType.NewTransaction,
+        RoutingStyle.gossip,
+        () => Promise.resolve(undefined),
+        () => Promise.resolve(undefined),
+      )
+    }
```

Comment:
Later, I assume we'll just undo this, remove the log when a message isn't handled, and handle the concept of "unknown" messages by increasing banscore inside of the peer network.

Reply:
Yep, I'm expecting we'll refactor all of this network code to be more specific to our protocol.